src/components/SubscriptionTable/SubscriptionTable.tsx

```diff
@@ -19,9 +19,12 @@ interface TypeCorrectedSubscription {
 }
 
 const printDate = (date: Date) => {
-  const month = date.getMonth() + 1;
+  const month = date.getUTCMonth() + 1;
+  const day = date.getUTCDate();
 
-  return `${date.getFullYear()}-${month < 10 ? `0${month}` : month}-${date.getDate()}`;
+  return `${date.getUTCFullYear()}-${month < 10 ? `0${month}` : month}-${
```

Comment:
should we think about using a library to format dates? It doesn't have to be now, but maybe a tech-debt story for the future?
Patternfly recommends https://date-fns.org/ or https://day.js.org/

Reply:
Yeah, I don't think that's a bad idea at all. I'm happy to do that in this story or come back later. Whichever is preferred